### PR TITLE
Update the conference page

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change

Update the conference page with the latest info 🎉 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/conference/index.markdown [hunk 1]
```diff
@@ -10,18 +10,24 @@ og_image: /images/conference/social.png
...
+The Home Assistant Conference is an online event to celebrate the community. It’s where the Home Assistant community will share their ideas, creations and major milestones. The event will take place on December 13, 2020.
 
-We're still working out the details, so check back on this page for updates.
+The conference will contain three different tracks: everyone, advanced users and developers. For each track we have 5-6 speakers to share their creations, deep-dive into certain topics or share their epic automations.
 
-## Open call for speakers
+Paulus Schoutsen, the founder of Home Assistant, will give the opening and closing keynote, aided by other Home Assistant developers.
 
-We're currently looking for people that want to speak at the conference. Talks will need to be either 15 or 30 minutes. You will stream them from your house during the event.
+After the talks, attendees will be able to attend various sessions about different topics like automations, lovelace and Q&A with Home Assistant developers.
```
Comment (MartinHjelmare) [inline source/conference/index.markdown]:
```suggestion
After the talks, attendees will be able to join various sessions about different topics like automations, Lovelace and Q&A with Home Assistant developers.
```

File: source/conference/index.markdown [hunk 2]
```diff
@@ -10,18 +10,24 @@ og_image: /images/conference/social.png
...
 
-Talks can be about things you created, like a great automation, an amazing Lovelace dashboard, or a cool new feature. It can also be about your experience with Home Assistant, like how it has improved peoples lives, how you use it for education or some other unique use case.
+## Attending
 
-_The deadline for submitting a talk was October 31, 2020._
+The Home Assistant Conference will be hosted on Hopin, an online conference platform. Tickets to attend will cost $1, which will cover the cost of the platform.
```
Comment (MartinHjelmare) [inline source/conference/index.markdown]: Should we link to hopin.to? Maybe link again to the tickets?

Comment (frenck) [inline source/conference/index.markdown]:
```suggestion
The Home Assistant Conference will be hosted on [Hopin](https://hopin.to/events/home-assistant-conference), an online conference platform. Tickets to attend will cost $1, which will cover the cost of the platform.
```

File: source/conference/index.markdown (hunk 1 above)
Comment (MartinHjelmare) [reply source/conference/index.markdown]: Didn't mean to change lovelace capitalization. Not sure of our standard there.

**Review by frenck — APPROVED** [review]

**Pull request merged**